apps/prairielearn/src/migrations/20231116100000_lti13_users__create.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS
+  lti13_users (
```

Comment:
Conventionally, even for join tables like this where we can uniquely identify a row with a combination of other rows, we still include an `id` primary key.

Reply:
Why? I think a composite primary key for this table is sane. In operation we're more likely to UPSERT it than DELETE and we're highly unlikely to refer to it by its `id` (as currently designed), only through JOINs.

I'd be OK changing the `UNIQUE` to `PRIMARY KEY` if that's clearer. I'm not sure if Postgresql adds anything more to the PRIMARY KEY other than a uniqueness and non-null constraint, which are already present.

Reply:
Changed to `PRIMARY KEY` in 4932db8a

Reply:
The main reason is that it's a super useful property to have a stable pointer to specific rows, namely for foreign keys but also for UI where we want to reference a specific entity in a hidden form field or in the URL. @mwest1066 may have some additional thoughts as to why this is nice; we've certainly made a conscious effort to build the PT data model such that all tables have an `id` column.

Reply:
I agree that we should have an `id` column as the primary key -- it's helpful for batched migrations, as an example.

Reply:
Understood. Reworked in 0367ecd1